src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -2011,6 +2022,9 @@ class Project: ADOSVTBase
                     }
                     else {
                        $controlResult.AddMessage([VerificationResult]::Passed, "No user found with admin roles in the project.")
+                       $controlResult.AdditionalInfoInCSV += "NumAdminGuests: $($inactiveUsersWithAdminAccess.count) ; ";
+                       $UserList = $inactiveUsersWithAdminAccess | ForEach-Object { $_.DisplayName +': '+ $_.PrincipalName} | select-object -Unique -First 10;
```

Comment:
Wouldn't $inactiveUsersWithAdminAccess.count be 0 and $inactiveUsersWithAdminAccess empty in passing scenario?
Please test all possible scenarios

Reply:
in Pass no need to check that so putting NA in Add_CSV